haiku-apps/haikuutils/haikuutils-r130~git.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A tool to manage teams and threads"
+DESCRIPTION="SystemManager shows all running teams and lets you explore various team objects \
+(images, threads, semaphores, etc.). It also generates stack traces of selected threads.
+provide stack trace of selected thread.
+
+Warning! This software is incomplete."
+HOMEPAGE="https://github.com/X547/HaikuUtils"
+COPYRIGHT="2020 X512"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="881fb8eede8867613feea409fb7f943e346d0839"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="53fba978c84c7e72e1c22850ae54269906f59f3e716f5c80c7aea18e9b351c8b"
+SOURCE_FILENAME="HaikuUtils-$srcGitRev-$portVersion.tar.gz"
+SOURCE_DIR="HaikuUtils-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	haikuutils$secondaryArchSuffix = $portVersion
+	cmd:PictureView
+	cmd:Services
+	cmd:SystemManager
+	cmd:TraverseApps
+	cmd:Users
+	"
+# MsgDump currently broken on 32bit
+if [ "$effectiveTargetArchitecture" = x86_64 ]; then
+	PROVIDES+="
+		cmd:MsgDump
+		"
+fi
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+Utils="PictureView Services SystemManager TraverseApps Users"
+if [ "$effectiveTargetArchitecture" = x86_64 ]; then
+	Utils+=" MsgDump"
+fi
+DeskbarUtils="Services SystemManager Users"
```

Comment:
There are meant to be in the menu, the other binaries are meant to be run in Terminal.